uber/templates/preregistration/attendee_donation_form.html

```diff
@@ -5,7 +5,7 @@
 <div class="masthead">
 </div>
 <div class="panel panel-default">
-{% if attendee.paid == c.NOT_PAID or attendee.overridden_price %}
```

Comment:
Hm. We could conceivably do `if attendee.paid != c.PAID_BY_GROUP` -- the confusing part is that people in groups technically have badge prices, so the page was printing out "you bought a badge for $40" even though they hadn't bought their own badge.